components/posts/ContentRenderer.tsx

```
@@ -0,0 +1,586 @@
+      return (
+        <p
+          className={`${baseClasses} text-white/80 leading-relaxed mb-6 text-lg`}
+        >
+          {item.content}
+        </p>
+      );
+
+    case "markdown":
+      return (
+        <div className={`${baseClasses} prose prose-invert max-w-none mb-6`}>
+          <div dangerouslySetInnerHTML={{ __html: item.content }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)